chainguard-dev/edu
# Fix issue-420 for image provenance verification.

## Type of change
Update cosign provenance verification due to cosign 2.0 being released.

### What should this PR do?
Updates the commands to work with version 2.0 of cosign.

### Why are we making this change?
Instructions are wrong for version 2.0.

### What are the acceptance criteria? 



### How should this PR be tested?
Try verifying with a new version of cosign, they fail.

One thing to think about is if we should have both instructions, or one overarching place where we say that we should be using cosign v2.0
